src/index.js

```diff
@@ -86,24 +86,30 @@ function format(options) {
   )
 
   const isCss = /\.(css|less|scss)$/.test(filePath)
+  const isHtml = /\.html$/.test(filePath)
+  const isTypeScript = /\.(ts|tsx)$/.test(filePath)
```

Comment:
`const isTypeScript = /\.tsx?$/.test(filePath)` what was wrong with this?

Reply:
Nothing, just copied over from [Prettier](https://github.com/prettier/prettier/blob/655fef1bf916361669aa926eb4ead875d6a2c993/src/options.js#L33-L43). Can change back if you want.